### ISSUE TITLE
Verify our Q(sqrt 5) equation against Hirzebruch's paper

### ISSUE THREAD

**Issue by edgarcosta** [edgarcosta/hilbertmodularforms#14]
The result for the graded ring of modular forms for QQ(sqrt(5)) is
Corollary 1 on the bottom of page 108 in 
Hirzebruch - Hilbert's Modular Group of the Field QQ(sqrt(5)) and the Cubic Diagonal Surface of Clebsch and Klein

**Comment by edgarcosta** [comment]
@BenKBreen, will you be sharing with us some code that verifies this?

**Comment by BenKBreen** [comment]
Angie found a paper that had the Fourier coefficients for some of the generators that Hirzebruch used. In terms of the code (I'm using a slightly older version), these are given as 

F := QuadraticField(5);
ZF<w> := Integers(F);
N := ideal<ZF | 1>;
prec := 80;
M := HMFSpace(F, N, prec);

B1 := Basis(M,[2,2]);
B3 := Basis(M,[6,6]);
B5 := Basis(M,[10,10]);
B20 := Basis(M,[20,20]);

// Generators for canonical ring
sig1 := B1[1];
sig3 := -2^7*B3[2];
sig5 := 2^12*(-17/145620*B5[2] + 1/1456200*B5[3]);
delta := -115240146944/415277590078125*B20[3] - 512/3737498310703125*B20[4];

These should satisfy the relation in Hirzebruch's paper given as

// Relations
delta^2 eq sig5*(3125*sig5^3 + 2000*sig5^2*sig3*sig1^2 + 256*sig5^2*sig1^5-900*sig5*sig3^3*sig1 -128*sig5*sig3^2*sig1^4 +16*sig3^4*sig1^3 +108*sig3^5);

**Comment by BenKBreen** [comment]
I may have been a little hasty in closing this issue. I'll try to get generators and relations using a version of Sam Tripp's canring.m code and then perform the change of variables to see if we find the same canonical ring.

**Comment by edgarcosta** [comment]
@BenKBreen, could you add a file with that?

**Comment by BenKBreen** [comment]
I added a file titled sqrt5-verification.m to the Dropbox where Sam S. was checking equations for sqrt2 and sqrt13. I might have to fiddle with the code a little bit to before uploading it to github

I was able to produce rational maps f : S -> S1 and g : S1  -> S between the surface computed from our code ( S ) and Hirzebruch's surface ( S1 ). The compositions f*g = g*f = Id which should fully close this issue.
